### PR TITLE
FISH-8142 Update Mojarra

### PR DESCRIPTION
**Pull request by Viii3**
Updating the Faces TCK runner to download and use version 4.0.3 of the TCK, which has no issues with Mojarra 4.0.7.

### CODE REVIEW

File: faces-tck/pom.xml-old-tck-run [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2021, 2022 Contributors to the Eclipse Foundation. All rights reserved.
+    Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation. All rights reserved.
```
Comment (breakponchito) [inline faces-tck/pom.xml-old-tck-run:4]: this should need to be 2024

**Comment by breakponchito** [comment]
Did you test this against Payara 6?

**Comment by Viii3** [comment]
It has been tested against both Payara 5 and Payara 6.

**Comment by breakponchito** [comment]
tested with success: https://jenkins.payara.fish/view/All/job/JakartaEE-10-TCK-Development-FISH-8142/

**Review by breakponchito — APPROVED** [review]
LGTM

**Pull request merged**